egs/tedlium/s5_r3/local/chain/run_tdnn_1c.sh

```diff
@@ -0,0 +1,249 @@
+#!/bin/bash
+
+# 1g is as 1f but moving to a factorized TDNN (TDNN-F) model, re-tuning it, and
+#  switching to unconstrained egs (the last of which gives around 0.1%
+#  improvement).  (Note: I don't believe the Tedlium TDNN models were,
+#  previously, very well-tuned).
+
+# local/chain/compare_wer_general.sh exp/chain_cleaned/tdnn1f_sp_bi exp/chain_cleaned/tdnn1g_sp
+# System                tdnn1f_sp_bi tdnn1g_sp  tdnn_1c
+# WER on dev(orig)            8.9       7.9      8.03
```

Comment:
It may not be clear from this, that the 1f and 1g were from the s5_r2 directory. (Is that the case?)
Use the compare_wer_general.sh script to compare 1b and 1c in this directory, and make sure the chain_dir_info.pl is up to date for this system.
It's disappointing that the extra data doesn't help.  Was this directory

Reply:
those numbers were in `s5_r2/local/chain/run_tdnn.sh run_tdnn.sh`
unfortunately I don't have old runs results 
the new numbers that I added, are up to date with this recipe (a recent run from chain_dir_info.pl)
I can run another recipe to have clean numbers for the old setups - should I run both 1f and 1g?

Reply:
You don't have to rerun the s5_r2 setup, but the currently-checked-in recipe in s5_r3 is what I want a comparison with.  And check that the tri3 results are similar to what is in the RESULTS file for tedlium s5_r3.

Reply:
sure, will run the original recipe of s5_r3 and share results (will start today, will take a few days)

Reply:
NN training is at 25% progress (I did a clean run from the very scratch) - will update the results once it's done

Reply:
```
dev:          %WER 8.15 [ 1449 / 17783, 241 ins, 302 del, 906 sub ]
dev_rescore:  %WER 7.69 [ 1367 / 17783, 237 ins, 323 del, 807 sub ]
test:         %WER 8.19 [ 2251 / 27500, 313 ins, 612 del, 1326 sub ]
test_rescore: %WER 7.77 [ 2137 / 27500, 303 ins, 572 del, 1262 sub ]
```

Reply:
so what do you think? should we proceed with the current PR?

Reply:
Yes.  Please use the compare_wer_general.sh to compare the two setups that you ran in s5_r3,and put that output at the top of run_tdnn_1c.sh, along with the chain_dir_info.pl from both runs.